backend/entities/oauth/base/service.go

```diff
@@ -0,0 +1,184 @@
+package base
+
+import (
+	"encoding/json"
```

Comment:
fast json marshalling/unmarshalling library we are using instead
```suggestion
		go_json "github.com/goccy/go-json"
```